rfcs/0140-simple-package-paths.md

```diff
@@ -0,0 +1,234 @@
+---
+feature: simple-package-paths
+start-date: 2022-09-02
+author: Silvan Mosberger
+co-authors: Nixpkgs Architecture Team
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/nixpkgs/pull/211832
+---
+
+# Summary
+[summary]: #summary
+
+Auto-generate trivial top-level attribute definitions in `pkgs/top-level/all-packages.nix` from a directory structure that matches the attribute name.
+This makes it much easier to contribute new packages packages, since there's no more guessing needed as to where the package should go, both in the ad-hoc directory categories and in `pkgs/top-level/all-packages.nix`.
```

Comment:
```suggestion
This makes it much easier to contribute new packages, since there's no more guessing needed as to where the package should go, both in the ad-hoc directory categories and in `pkgs/top-level/all-packages.nix`.
```